### DISCUSSION TITLE
Row, ResponsiveRow, Column, ListView: add new formatting properties

### DISCUSSION THREAD

**Discussion by ClearSafety** [flet-dev/flet#4142] [Ideas]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the requested feature

When one needs to work with Row, ResponsiveRow, Column or ListView and would like to display it with certain configuration, it's necessary to use a Container. For example: if you wish a border, you need a Container, because these classes don't have this feature.
So, I'd like to suggest to these controls be included properties like: border, border_radius, shadow, bgcolor.

**Comment by ndonkoHenri** [comment]
Sorry, but not all Controls can have these props.

Feel free to open an issue if you face any issues when wrapping them with a `Container`.
